src/Sylius/Bundle/OrderBundle/Twig/AggregateAdjustmentsExtension.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\OrderBundle\Twig;
+
+use Sylius\Bundle\OrderBundle\Aggregator\AdjustmentsAggregatorInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class AggregateAdjustmentsExtension extends \Twig_Extension
```

Comment:
We should create a templating helper and inject it here, like we do for rest of Twig Extensions https://github.com/Sylius/Sylius/tree/master/src/Sylius/Bundle/SettingsBundle/Templating/Helper.